apps/backend/prisma/schema.prisma

```diff
@@ -874,3 +880,32 @@ model SubscriptionInvoice {
   @@id([tenancyId, id])
   @@unique([tenancyId, stripeInvoiceId])
 }
+
+model OutgoingRequest {
+  id String @id @default(uuid()) @db.Uuid
+
+  createdAt DateTime @default(now())
+
+  qstashOptions Json
+  fulfilledAt   DateTime?
+
+  @@index([fulfilledAt])
```

Comment:
maybe we can rename this to "startedFulfillingAt" just to make it clearer for the future